content/SCALE/SCALETutorials/Shares/SMB/_index.md

```diff
@@ -241,98 +258,93 @@ See [Permissions]({{< relref "PermissionsScale.md" >}}) for more information on
 
 {{< include file="/static/includes/UsingTraversePermission.md" >}}
 
-## Starting the SMB Service
+## Starting the SMB 
```

Comment:
Why remove service here? It feels awkward this way.

Reply:
oh that was an accident.